conf/nginx/sites-available/off

```diff
@@ -13,6 +13,21 @@ server {
 	}
 }
 
+# map to decide if we go to the main (8004) or secondary service (8005)
```

Comment:
"secondary" sounds like it's not a priority. I almost expected that we would send slow requests that we care a bit less about to the "secondary" Apache.

I think it would be clearer to use a name that gives the intent, like "fasttrack" or "fastlane".